contributors/design-proposals/storage/csi-migration.md

```diff
@@ -841,6 +841,43 @@ When the above conditions are not met, the PV Controller will use FindProvisiona
 to determine the in-tree plugin that can be used for provisioning a volume (as is
 the case today).
 
+Update: 1/13/2020
+
+In Beta we discovered issue: https://github.com/kubernetes/kubernetes/issues/79043
+
+In order to resolve this the design needs to be modified. When migration is "on"
+the Persistent Volume Controller will still set the Storage Provisioner
+annotation (volume.beta.kubernetes.io/storage-provisioner) with the name of the
+migrated CSI driver. However, it will also set an additional annotation
+`volume.beta.kubernetes.io/migrated-to` to the CSI Driver name. Additionally,
+the PV Controller will be modified so that it lists all PV's and PVC's on
+startup and checks the `volume.beta.kubernetes.io/storage-provisioner` and
+`pv.kubernetes.io/provisioned-by` annotations respectively to anneal the
```

Comment:
In this context, I think you can replace "anneal the required" with "set the correct"